healthcheck/checker.go

```diff
@@ -36,22 +34,12 @@ func FromChecks(config config.Config, checks []models.Healthcheck) checker {
 }
 
 func (c checker) IsHealthy(ctx context.Context) bool {
-	appLogger := logger.Get(ctx)
-
-	// Custom logger to discard Philae output
```

Comment:
And we don't need to discard Philae output with this version 4.3.0?

Reply:
`Because there is no more output with Philae, everything gets in the prober.ProberResult method`